tests/unit/modules/test_select.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+import pandas as pd
+import pytest
+import xarray as xr
+
+from pywatts.modules.preprocessing.select import Select
+
+
+class TestSelect(unittest.TestCase):
+    def setUp(self) -> None:
+        self.select = Select(start=1, stop=4, step=1)
+
+    def tearDown(self) -> None:
+        self.select = None
+
+    def test_get_params(self):
+        params = self.select.get_params()
+
+        self.assertEqual(params,
+                         {
+                             "start": 1,
+                             "stop": 4,
+                             "step": 1,
+                         })
+
+    def test_set_params(self):
+        self.assertEqual(self.select.get_params(),
+                         {
+                             "start": 1,
+                             "stop": 4,
+                             "step": 1,
+                         })
+        self.select.set_params(start=3, stop=20, step=12)
+        self.assertEqual(self.select.get_params(),
+                         {
+                             "start": 3,
+                             "stop": 20,
+                             "step": 12,
+                         })
+
+    def test_transform(self):
```

Comment:
Are there more edge cases like start=stop or step larger then array?

Reply:
Or only stop is set?

Reply:
I add a check method that checks the type of start and if start is smaller than stop. 

The step can be larger then an array. E.g. `slice(0,2, 100000000000000)` would only take the value at position 0.